SampleApp/www/js/index.js

```diff
@@ -35,6 +35,27 @@ var app = {
     // deviceready Event Handler
     onDeviceReady: function() {
         this.receivedEvent('deviceready');
+
+        cordova.plugins.ForeSeeAPI.start(this.onSuccess);
```

Comment:
@sleeke unless I'm missing something, this has the same effect, since the code was in `receivedEvent` which is called by `onDeviceReady` (`this.receivedEvent('deviceready');`). However, I remember thinking this was pretty goofy when I first saw it, anyway, so I think this is a good change. But in this case, can we just remove `receivedEvent` entirely? It doesn't look like it's doing anything.

Reply:
Yup I don't see why not. I'll add a fresh commit for that change

Reply:
✅